src/nvim/api/ui_events.in.h

```diff
@@ -41,6 +41,10 @@ void option_set(String name, Object value)
   FUNC_API_SINCE(4);
 void chdir(String path)
   FUNC_API_SINCE(12);
+void cursor_on(void)
+  FUNC_API_SINCE(13);
+void cursor_off(void)
+  FUNC_API_SINCE(13);
```

Comment:
Why are these two needed? Don't `busy_stop` and `busy_start` do the same thing?

Reply:
Yes, after a try it works.  I'm still not familiar with the tui part 😢. So I tried sending invisible/normal. will fix

Reply:
oh i found it `flush_buf_end` it will check the `tui->is_invisible && !should_invisible(tui)` to show or hide cursor